components/Carousel/Carousel.js

```diff
@@ -1,6 +1,57 @@
-class Carousel {
+  class Carousel {
+     constructor(element) {
+		  this.element = element;
+      this.carouselitems = element.querySelectorAll(".Carousel__item");
```

Comment:
Try to consistently keep your code aligned and indented appropriately.  It makes it much easier for someone else to scan your code to find something.  It'll help when you need to go back through your code also.